.github/workflows/cache-issues.yaml

```diff
@@ -8,10 +8,15 @@ jobs:
   collect_n_upload:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            .github/dump_issues.py
+          sparse-checkout-cone-mode: false
       - run: |
           mkdir -p issues
           for repo in scylladb scylla-enterprise scylla-manager scylla-operator scylla-cluster-tests scylla-dtest qa-tasks scylla-tools-java ; do
-            gh issue list --state all --json number,state,labels --limit 30000 --template '{{range .}}{{.number}},{{.state}},{{range .labels}}{{.name}}|{{end}}{{println ""}}{{end}}' --repo scylladb/$repo > issues/scylladb_$repo.csv
+            .github/dump_issues.py --repository scylladb/$repo
```

Comment:
why not to collect PR info like with issues (same command, just replace `gh issue list` with `gh pr list`)?
AFAIK graphql is very limited in terms of requests quota.

Reply:
It would mean we'll need to change the client code, both in SCT and dtest.

But I didn't look at the cost, and since we cut most of the problematic call away, we have plenty of it.

I'll need to add the costs and see how much it is per repository

Reply:
I've checked the costs, and it was counting each page as one of the 500 limit

doing as you suggested seems to be 2 per repository: 2 X 8 = 16

with graphql it's:  number of pages X 8 repos, and each repo has different number of pages, and the limit per page it 100 items in graphql

so I'll ditch the graphql for now